src/main/java/io/debezium/connector/yugabytedb/YugabyteDBErrorHandler.java

```diff
@@ -23,6 +24,15 @@ public YugabyteDBErrorHandler(YugabyteDBConnectorConfig connectorConfig, ChangeE
 
     @Override
     protected boolean isRetriable(Throwable throwable) {
+        // Error throwable class indicating tablet split
```

Comment:
For my understanding, is this what triggers a restart?

Reply:
Yes, this restarts the task. But we are not doing the connector restart now, the tablet split is handled gracefully without a restart.